src/test/java/core/basesyntax/ReaderServiceTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReaderServiceTest {
+    private static final String correctFilePath = "src/main/resources/inputFile.csv";
```

Comment:
You should create separate file for tests and store it in test resources directory.

Reply:
`src/test/resources/`